### DISCUSSION TITLE
PhantomJs

### DISCUSSION THREAD

**Discussion by 111100001** [bibanon/tubeup#289] [Q&A] [answered]
![image](https://github.com/bibanon/tubeup/assets/24229327/092c5e6c-31d8-4889-ba82-af461db3c313)

this message started appearing two days ago. i tried to install phantomjs but none of the available versions are  compatible with my vm, building phantomjs from source code failed also. is this fine?

**Comment by mrpapersonic** [comment, marked as answer]
This is a problem with yt-dlp. It doesn't look like it affects actual downloading (aside from throttling), so it's just an info message telling you your download might be throttled by YouTube.

**Comment by vxbinaca** [comment]
@mrpapersonic one alternative is since yt-dlp defaults to `--best-video` and `--best-audio-` but we call those explicitly, would allowing the default formats perhaps fix this?

The problem will only grow as time goes on.

**Comment by vxbinaca** [comment]
this was fixed by an update to yt-dlp if I recall
